### PR TITLE
PA-1367: Bring down the KVDB nodes  during restore.

### PR DESCRIPTION
**Pull request by jkagliwal-px**
Scenario - 

Steps -

1. Deploy multiple data services having volume replicas more than one.
2. Start a long running workload on all the data services.
3. Perform multiple backups.
4. Perform restores and bring down the KVDB .
5. Perform following validation for each restored DS:
a. Application data 
b. Resource specification
c. Application configuration
d. PDS Control plane UI should be able to detect them.
e. Existing data service should be intact
f. Restored data service endpoint should be accessible and in term workload should succeed on the restored DS. Delete all the restored data services.

Aetos Link - https://aetos.pwx.purestorage.com/resultSet/testSetID/331279
JenkinsRun - https://jenkins.pwx.dev.purestorage.com/job/PDS/job/pds-branch-build-test/497/console

### CODE REVIEW

File: tests/pds/pds_resiliency_test.go [hunk 1]
```diff
@@ -1197,3 +1197,148 @@ var _ = Describe("{RestoreDSDuringPXPoolExpansion}", func() {
...
+					log.InfoD("Number of backups to be taken are- %v", len(deps))
+					dsEntity = restoreBkp.DSEntity{
+						Deployment: deployment,
+					}
+				})
+			}
+		})
+		Step("Running Workloads before taking backups", func() {
+			for _, pdsDeployment := range deps {
+				ckSum, wlDep, err := dsTest.InsertDataAndReturnChecksum(pdsDeployment, wkloadParams)
```
Comment (lsrinivas-pure) [inline tests/pds/pds_resiliency_test.go:1250]: validate the err here

File: tests/pds/pds_resiliency_test.go [hunk 2]
```diff
@@ -1197,3 +1197,148 @@ var _ = Describe("{RestoreDSDuringPXPoolExpansion}", func() {
...
+			}
+
+		})
+		Step("Trigger restore during KVDB pods are down", func() {
+			for _, deployment := range deployments {
+				failuretype := pdslib.TypeOfFailure{
+					Type: RestoreDSDuringKVDBFailOver,
+					Method: func() error {
+						return KillKvdbMasterNodeAndFailover()
+					},
```
Comment (lsrinivas-pure) [inline tests/pds/pds_resiliency_test.go:1272]: this will kill master node once or do you want to try killing all kvdb nodes?

Comment (jkagliwal-px) [reply tests/pds/pds_resiliency_test.go:1272]: we want to try delete all while restore is running

File: drivers/pds/parameters/pds_default_parameters.json [hunk 3]
```diff
@@ -2,108 +2,40 @@
...
+    "AccountName": "Automation",
     "TenantName": "Default",
     "ProjectName": "Default",
     "ClusterType": "onprem",
     "Namespace": "automation",
-    "PxNamespace": "portworx",
+    "PxNamespace": "kube-system",
     "PDSNamespace": "pds-system"
   },
-  "CleanUpParams": {
```
Comment (Madanagopal19) [inline drivers/pds/parameters/pds_default_parameters.json:102]: Remove changes from this file.

**Review by Madanagopal19 — APPROVED** [review]
Approving with a minor comment

**Review by aravindranath14 — APPROVED** [review]
LGTM

**Review by lsrinivas-pure — APPROVED** [review]
LGTM

**Pull request merged**